helm_deploy/hmpps-registers-to-delius-update/values.yaml

```diff
@@ -1,5 +1,55 @@
 ---
 # Values here are the same across all environments
 
+generic-service:
+  nameOverride: hmpps-registers-to-delius-update
+
+  image:
+    repository: quay.io/hmpps/hmpps-registers-to-delius-update
+    tag: app_version    # override at deployment time
+    port: 8080
+
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: "nginx"
+      nginx.ingress.kubernetes.io/custom-http-errors: "418"
+    tlsSecretName: hmpps-registers-to-delius-update-cert
+    path: /
+
+  env:
+    SERVER_PORT: 8080
+    JAVA_OPTS: "-Xmx512m"
+    APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
+    SPRING_PROFILES_ACTIVE: "logstash"
+
+  # Pre-existing kubernetes secrets to load as environment variables in the deployment.
+  # namespace_secrets:
+  #   [name of kubernetes secret]:
+  #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
+
+  namespace_secrets:
+    hmpps-registers-to-delius-update:
+      APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+      HMPPS_AUTH_CLIENT_CLIENT_ID: "HMPPS_AUTH_CLIENT_CLIENT_ID"
```

Comment:
this should be 
```
      HMPPS_AUTH_CLIENT_CLIENT_ID: "OAUTH_CLIENT_ID"
```
and same with one below

Reply:
yes - you're right